src/TitleBuilder.php

```diff
@@ -179,6 +179,7 @@ private function cleanTitleSegment( $segment ) {
 		$segmentParts = array_map( 'trim', $segmentParts );
 		$segment = implode( ', ', $segmentParts );
 		$segment = str_replace( ' ', '_', $segment );
+		$segment = str_replace( ':', '_', $segment );
```

Comment:
What's the reason for this change? Technically subpage titles _can_ contain colons.